src/go/k8s/pkg/console/admin.go

```diff
@@ -90,6 +90,7 @@ func NewKafkaAdmin(
 	go func() {
 		<-ctx.Done()
 		admClient.Close()
+		kclient.Close()
```

Comment:
One question, if the call to close is done on the adminClient, and the admClient takes in as a client the kclient, why isnt the closing function asking that the kclient also close?

Reply:
```
func (cl *Client) Close()
    Close closes the underlying *kgo.Client.```

Reply:
What's the evidence for a leak, and that this PR will fix it? admClient.Close does close the underlying kgo.Client.

Reply:
You are right @alejandroEsc.

@twmb the https://github.com/redpanda-data/redpanda/issues/12533. Operator opens per reconciliation kgo client and closes it. The number of active connection only grows. One operator is getting OOM, then number of active connections in Redpanda drops and the claiming # connections starts again. I will open PR to instrument closing function as I don't believe it works as expected.

Reply:
Is the context truly canceled?